docs/releasing.md

```diff
@@ -1,8 +1,36 @@
-## Release guide for maintainers
+### Release guide for maintainers
+
+## Release Policy
+
+### Cadence
+
+We ship twice a month, on the Thursday before the 15th and the Thursday before 1st. These dates are added to the internal Lighthouse calendar.
```

Comment:
Also- it would be helpful to have something in here about how we choose to number our releases (i.e. why 2.7 vs 2.7.1)

Reply:
it's in there. see `Versioning`

Reply:
Ah, I missed that. SGTM.